persistently-store-data-using-Ballerina/frontend_server/src/layouts/dashboard/side-nav-item.js

```diff
@@ -0,0 +1,110 @@
+/**
```

Comment:
This file should be in the `components` directory.

Reply:
updated